### PR TITLE
Switch to factory methods

### PR DESCRIPTION
**Pull request by siy**

### CODE REVIEW

**Review by Sajjon — APPROVED** [review]
LGTM

File: radixdlt/src/main/java/com/radixdlt/consensus/NewView.java [hunk 1]
```diff
@@ -68,7 +67,7 @@
 		@JsonProperty("committedQC") QuorumCertificate committedQC,
 		@JsonProperty("signature") ECDSASignature signature
 	) throws CryptoException {
-		this(BFTNode.create(new ECPublicKey(author)), view != null ? View.of(view) : null, qc, committedQC, signature);
+		this(BFTNode.fromPublicKeyBytes(author), view != null ? View.of(view) : null, qc, committedQC, signature);
```
Comment (clstrfsck) [inline radixdlt/src/main/java/com/radixdlt/consensus/NewView.java:70]: Snuck that one in.  Nice.

**Review by clstrfsck — APPROVED** [review]
Looks good.

File: radixdlt/src/main/java/com/radixdlt/consensus/NewView.java (hunk 1 above)
Comment (siy) [reply radixdlt/src/main/java/com/radixdlt/consensus/NewView.java:70]: Yeah, repetitive code...

**Pull request merged**